rules/docker-ptf.mk

```diff
@@ -0,0 +1,6 @@
+# docker image for ptf
+
+DOCKER_PTF = docker-ptf.gz
+$(DOCKER_PTF)_PATH = $(DOCKERS_PATH)/docker-ptf
+$(DOCKER_PTF)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF)
+SONIC_DOCKER_IMAGES += $(DOCKER_PTF)
```

Comment:
This is a recipe for building a docker image
It defines name of the image, path to dockerfile and debian packages that it needs to install